### ISSUE TITLE
Read More: Last line of multi-line text has been truncated.

### ISSUE THREAD

**Issue by aalam** [pg-irc/pathways-frontend#733]
**Describe the bug**
Last line of multi-line text has been truncated. Lower portion of text is not visible properly. It may not be bug, only enhancement as contents are available to user by clicking 'Read more'.

**App version**
Staging 1.0.24

**To Reproduce**
Application Language - Punjabi
1. Go to 'Home'
2. Click on 'Apply for a driver's Licensee
3.  Check line above 'Read more' button
5. This applies to all topics

Application Language - English
1. Go to 'Home'
2. Scroll down to Health Care'
2. Click on 'Medical Insurance costs and Financial assistance
3.  Check line above 'Read more' button
4. See 'From 'MSP'
5. This applies to all topics

**Expected behavior**
Text should not appear to be cut off.

**Observed behavior**
Text looks like its been cut off

**Screenshots**
Attached, Highlighted part.

**Smartphone (please complete the following information):**
 - Device: Samsung Note 9 and Apple iPhone
 - OS: Android 9 / iOS 12.3.1
![IMG_0679](https://user-images.githubusercontent.com/600806/61172316-59ce7e00-a537-11e9-8254-13b32d34629d.PNG)

**Additional context**
This is minor issue as complete text available when click on 'Read More' 
![20190706_144314](https://user-images.githubusercontent.com/600806/61172140-60f48c80-a535-11e9-88c5-c41f6d718b94.jpg)

**Comment by ArtiBPG** [comment]
@cherrielam review and suggest a fix. Can we use fade out like this: https://screencast.com/t/QPc9UhZB

**Comment by ArtiBPG** [comment]
Should be addressed after #719

**Comment by cherrielam** [comment]
Agree — proposed fix for the Read More component would be to have a gradient/fade out effect as per: https://zpl.io/aByGdBm

![Screen@1x](https://user-images.githubusercontent.com/21990736/61419626-e8365d00-a8b3-11e9-808c-650fdee78dae.png)

**Comment by rasmus-storjohann-PG** [comment]
Closing this issue. Ian has had a go and given up. Ben has had several goes and given up, this is from Ben: "we've had many variations of that exact ticket and I've written many variations of responses. The way it works is the best we've been able to do".

**Comment by rasmus-storjohann-PG** [comment]
Closing this issue. Ian has had a go and given up. Ben has had several goes and given up, this is from Ben: "we've had many variations of that exact ticket and I've written many variations of responses. The way it works is the best we've been able to do".

There is no point revisiting this issue until there is improved support in thirdparty libraries that let us solve this in a better way.

**Comment by rasmus-storjohann-PG** [comment]
Closing this issue. Ian has had a go and given up. Ben has had several goes and given up, this is from Ben: "we've had many variations of that exact ticket and I've written many variations of responses. The way it works is the best we've been able to do".

There is no point revisiting this issue until there is improved support in thirdparty libraries that let us solve this in a better way.
